tls/s2n_record.h

```diff
@@ -55,37 +56,37 @@
  * 1024 bytes of encryption expansion and padding.
  * Since S2N does not support compression, we can ignore the compression overhead.
  */
-#define S2N_TLS12_ENCRYPTION_OVERHEAD_SIZE      1024
-#define S2N_TLS12_MAX_RECORD_LEN_FOR(frag)      ((frag) + S2N_TLS12_ENCRYPTION_OVERHEAD_SIZE \
-                                                        + S2N_TLS_RECORD_HEADER_LENGTH)
-#define S2N_TLS12_MAXIMUM_RECORD_LENGTH         S2N_TLS12_MAX_RECORD_LEN_FOR(S2N_TLS_MAXIMUM_FRAGMENT_LENGTH)
+#define S2N_TLS12_ENCRYPTION_OVERHEAD_SIZE 1024
+#define S2N_TLS12_MAX_RECORD_LEN_FOR(frag) ((frag) + S2N_TLS12_ENCRYPTION_OVERHEAD_SIZE \
+        + S2N_TLS_RECORD_HEADER_LENGTH)
+#define S2N_TLS12_MAXIMUM_RECORD_LENGTH S2N_TLS12_MAX_RECORD_LEN_FOR(S2N_TLS_MAXIMUM_FRAGMENT_LENGTH)
```

Comment:
Why aren't these aligned? Is it because one is a multiline define?

Reply:
Yes.